### ISSUE TITLE
Base code questions in raisimGymtorch

### ISSUE THREAD

**Issue by LALALADA-SPONGEBOB** [raisimTech/raisimLib#356]
Hi, raisimTech!
I've been reading and trying to understand the raisimGymtorch code, but found something that I'm not sure about.

1. In VectorizedEnvironment.hpp, Normalsampler.seed(), 

`  void seed(int seed) {
    // this ensures that every thread gets a different seed
#pragma omp parallel for schedule(static, 1)
     for (int i = 0; i < THREAD_COUNT; i++)
       normal_[0].seed(i + seed);
  }`

I think the code should be normal_[i].seed(i+seed) for threads to have different seeds, or am I misunderstanding something in the code?

2. Also in runners.py code,

`actor.update()
 actor.distribution.enforce_minimum_std((torch.ones(12)*0.2).to(device))`

Is there any reason that actor.update() comes before actor.distribution.enforce_minimum_std((torch.ones(12)*0.2).to(device))?

Best.

**Comment by jhwangbo** [comment]
1. That seems like my mistake. I'll fix it.
2. I don't think this order makes any meaningful difference. Do you see any issue?

**Comment by LALALADA-SPONGEBOB** [comment]
Thank you for the reply.
For second question, I thought if actor.update() comes first then the collected data are from distribution with std that is not clipped by minimum_std condition. I thought in this way because data collecting distribution has std which is 'std_np' memebr variable in 'class MultivariateGaussianDiagonalCovariance'

**Comment by jhwangbo** [comment]
I see. Not updating std_np can be an issue. I think the better way is to update std_npp in ``enforce_minimum_std``. I'll implement this change as well.

**Comment by LALALADA-SPONGEBOB** [comment]
Thank you :)
